tests/metrics/test_randomisation_metrics.py

```diff
@@ -482,6 +492,7 @@ def test_random_logit(
         a_batch=a_batch,
         **call_params,
     )
+    print("scores!!", scores)
```

Comment:
Please remove

Reply:
Also a general remark: knowing why the test has failed is often very valuable and the print ⁸statement is a symptom of the missing information in the following assert statement.

Although `assert all(s == expected for s in scores)` is indeed very elegant, its error message lacks any information why it failed.

It could be worth writing a small test helper function to compare scores against expected values and output the respective erroneous scores. This helper function could could be used in nearly all metric test functions. But this should be tackled in a separate PR and is not in this scope here of course.

Reply:
Very valid points!